changelog/0015-separate-celery-datadog-http-check.yml

```diff
@@ -0,0 +1,22 @@
+title: Separate celery datadog http check
+key: separate-celery-datadog-http-check.yml
+date: 2019-02-22
+optional_per_env: yes
+min_commcare_version:
+max_commcare_version: 81325fbd9e3131c710179f7246dbe9caccb45154
```

Comment:
Just confirming that it is `max_commcare_version`, and not `min_commcare_version`, that means "this change should be applied from this revision number onwards".

I might be misunderstanding, but it looks like `0005-support-multiple-kafak-brokers.yml`, `0007-reorganize-pillows.yml`, `0008-blob-metadata-part-1.yml`, and `0009-blob-metadata-part-2.yml` use `min_commcare_version` to mean this, and `0004-add-analytics-queue.yml`, `0006-new-case-importer-celery-queue.yml`, and `0012-generalize-load-case-from-fixture.yml` use `max_commcare_version` to mean the same thing. ... Or are 0005, 0007, 0008 and 0009 NOT applied after their given revision numbers?

Reply:
This part of the docs template for these changes is in my mind the best explanation for the difference: https://github.com/dimagi/commcare-cloud/blob/3ba62c0e12ead0147daf4ca3c77afcfe62f4c900/src/commcare_cloud/manage_commcare_cloud/changelog.md.j2#L13-L21. I myself refer back to that whenever I'm unsure